ts/src/bybit.ts

```diff
@@ -2765,11 +2775,25 @@ export default class bybit extends Exchange {
         }
         const marketId = this.safeString (trade, 'symbol');
         market = this.safeMarket (marketId, market, undefined, 'spot');
+        const symbol = market['symbol'];
```

Comment:
actually right now I'm not sure this is running on spot trades 
![image](https://github.com/ccxt/ccxt/assets/43336371/86ca5a59-da4f-43e0-b05b-b2f7b9d4cbaf)

Reply:
yes, now I see that in case of spot trading (not only private but also public) we will get into `parseContractTrade`